CMakeLists.txt

```
@@ -63,8 +64,29 @@ configure_file("${CMAKE_SOURCE_DIR}/cmake/templates/license.h.in" include/licens
 
 include(EthOptions)
 configure_project(TESTS)
+set(LATEST_Z3_VERSION "4.8.12")
+set(LOWEST_Z3_VERSION_SUPPORTED "4.8.0")
```

We could shorten this name a bit:
```suggestion
set(MINIMUM_Z3_VERSION "4.8.0")
```

---

done